gallery/workbook/Azure Security Center.json

```diff
@@ -24,6 +24,12 @@
 					"name": "Vulnerability Assessment Findings",
 					"description": "View the findings of vulnerability scans of your Azure resources"
 				},
+				{
+					"id": "Workbooks/Azure Security Center/Active Alerts",
+					"author": "Microsoft",
+					"name": "Active alerts",
+					"description": "View the the alerts of your Azure resources"
```

Comment:
Change to: View the active security alerts of your Azure resources.

Reply:
Changed the description message to the suggested one.